.travis.yml

```diff
@@ -8,6 +8,11 @@ matrix:
     env:
     - LINT=true
     - COVERAGE=true
+  # Adding ppc64le jobs
+  - go: "tip"
+    arch: ppc64le
+    env:
+    - COVERAGE=true
```

Comment:
We cannot use COVERAGE in this step, because it will double-report it to codecov.

You can introduce another var TEST=true and add this under `script:`
```
if [ "$TEST" == true ]; then make test ; fi
```

Reply:
Actually, don't need new variable at all, tests are run unconditionally already. So just remove the env var.

Reply:
Thank you for suggesting the changes. I had removed the COVERAGE env while running go tip  on ppc64le